### ISSUE TITLE
Textmate setting scope

### ISSUE THREAD

**Issue by doodleslice** [emmetio/Emmet.tmplugin#12]
Is there a place to set Emmet's scope in Textmate?  Most bundle commands let you set a scope selector so that the command only works in certain document types. This helps reduce keyboard shortcut conflicts.

**Comment by sergeche** [comment]
No, right now it’s not possible due to nature of Emmet commands, but I’ll think about it
